docs/reference/data-streams/downsampling-ilm.asciidoc

```diff
@@ -0,0 +1,548 @@
+[[downsampling-ilm]]
+=== Run downsampling with ILM
+++++
+<titleabbrev>Run downsampling with ILM</titleabbrev>
+++++
+
+preview::[]
+
+This is a simplified example that allows you to see quickly how
+<<downsampling,downsampling>> works as part of an ILM policy to reduce the
+storage size of a sampled set of metrics. The example uses typical Kubernetes
+system monitoring data. To test out downsampling with ILM, follow these steps:
+
+. Check the <<downsampling-ilm-prereqs,prerequisites>>.
+. <<downsampling-ilm-policy>>.
+. <<downsampling-ilm-create-index-template>>.
+. <<downsampling-ilm-ingest-data>>.
+. <<downsampling-ilm-view-results>>.
+
+[discrete]
+[[downsampling-ilm-prereqs]]
+==== Prerequisites
+
+Refer to <<tsds-prereqs,time series data stream prerequisites>>.
+
+Before running this example you may want to try the
+<<downsampling-manual,Run downsampling manually>> example.
+
+[discrete]
+[[downsampling-ilm-policy]]
+==== Create an index lifecycle policy
+
+Create an ILM policy for your time series data. While not required, an ILM
+policy is recommended to automate the management of your time series data stream
+indices.
+
+To enable downsampling, add a <<ilm-downsample,Downsample action>> and set
+<<ilm-downsample-options,`fixed_interval`>> to the frequency at which you want
+to aggregate the original time series data.
+
+In this example, an ILM policy is configired for the `hot` data tier. A
+downsample process runs after the first document is inserted into the index, and
+then repeats hourly.
+
+////
+[source,console]
+----
+PUT /_snapshot/found-snapshots
+{
+ "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+----
+// TESTSETUP
+////
+
+[source,console]
+----
+PUT _ilm/policy/datastream_policy   
+{
+  "policy": {                       
+    "phases": {
+      "hot": {                      
+        "actions": {
+          "rollover": {             
+            "max_docs": 1
+          },
+          "downsample": {
+  	          "fixed_interval": "1h"
+  	      }
+        }
+      }
+    }
+  }
+}
+----
+
+[discrete]
+[[downsampling-ilm-create-index-template]]
+==== Create an index template
+
+This creates an index template for a basic data stream. The available parameters
+for an index template are described in detail in <<set-up-a-data-stream,Set up a
+time series data stream>>.
+
+For simplicity, in the time series mapping all `time_series_metric` parameters
+are set to type `gauge`, but <<time-series-metric,other values>> such as
+`counter` and `histogram` may also be used. The `time_series_metric` values
```

Comment:
`histogram` is not supported. We only support `gauge` and `counter` metric types

Reply:
Changed to `For simplicity, in the time series mapping all `time_series_metric` parameters
are set to type `gauge`, but the `counter` metric type may also be used.` (via [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77))